testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -215,11 +219,22 @@ impl LocalSwarm {
             .collect::<Result<HashMap<_, _>>>()?;
 
         // We print out the root key to make it easy for users to deploy a local faucet
-        let encoded_root_key = hex::encode(root_key.to_bytes());
+        let encoded_root_key = EncodingType::Hex.encode_key("root_key", &root_key)?;
         info!(
             "The root (or mint) key for the swarm is: 0x{}",
-            encoded_root_key
+            String::from_utf8_lossy(encoded_root_key.as_slice())
         );
+        let root_key_path = dir_actual.as_ref().join("root_key");
```

Comment:
nit: could we move these into a helper function? e.g., `write_root_key_to_file()`?